django_irods

```diff
@@ -1 +1 @@
-Subproject commit 64cc56f3feb2617fc2fff1f3256e56ff96da319d
+Subproject commit 5a0359d72f2b4ec3a43e38a2be22d9ed12f092b0
```

Comment:
You cannot exclude it per se, but you can/should make sure you have the latest version of the master branch checked out for it.

It is a git repository like any other, so you can:
```bash
# From within the main level of the hydroshare repository
cd hs_docker_base
git fetch origin
git checkout master
git pull
cd ../
```

Check **git status**, and if the file is denoted as **modified**, then `git add hs_docker_base` and commit the change.